src/bundles/analytics.js

```diff
@@ -276,7 +288,8 @@ const createAnalyticsBundle = ({
       ACTIONS.ANALYTICS_DISABLED,
       ACTIONS.ANALYTICS_DISABLED,
       ACTIONS.ANALYTICS_ADD_CONSENT,
-      ACTIONS.ANALYTICS_REMOVE_CONSENT
+      ACTIONS.ANALYTICS_REMOVE_CONSENT,
+      ACTIONS.ANALYTICS_ASK_TO_ENABLE
```

Comment:
```suggestion
      ACTIONS.SET_DISPLAY_ANALYTICS_DISPLAY_BANNER
```
I think this name would make better sense as "SET_DISPLAY_ANALYTICS_DISPLAY_BANNER," what do you think? we're not technically asking users to enable metrics, because we're enabling them by default.

Reply:
updates to clearer names to match the refactor have been pushed